cypress/integration/data-import/make-fields-required-in-invoice-map-profile.spec.js

```diff
@@ -0,0 +1,83 @@
+import permissions from '../../support/dictionary/permissions';
+import TestTypes from '../../support/dictionary/testTypes';
+import DevTeams from '../../support/dictionary/devTeams';
+import TopMenu from '../../support/fragments/topMenu';
+import FieldMappingProfiles from '../../support/fragments/data_import/mapping_profiles/fieldMappingProfiles';
+import NewMappingProfile from '../../support/fragments/data_import/mapping_profiles/newMappingProfile';
+import Helper from '../../support/fragments/finance/financeHelper';
+import Users from '../../support/fragments/users/users';
+import MappingProfileDetails from '../../support/fragments/data_import/mapping_profiles/mappingProfileDetails';
+
+describe('ui-data-import: Make some of the fields on the Invoice field mapping profile required', () => {
+  let user = null;
+  const mappingProfileName = `C343284 invoice mapping profile ${Helper.getRandomBarcode()}`;
+
+  before(() => {
+    cy.createTempUser([
+      permissions.moduleDataImportEnabled.gui,
+      permissions.settingsDataImportEnabled.gui,
+      permissions.viewOrganization.gui
+    ])
+      .then(userProperties => {
+        user = userProperties;
+
+        cy.login(user.username, user.password, { path: TopMenu.mappingProfilesPath, waiter: FieldMappingProfiles.waitLoading });
+        cy.getAdminToken();
+      });
+  });
+
+  after(() => {
+    Users.deleteViaApi(user.userId);
+  });
+
+  it('C343284 Make some of the fields on the Invoice field mapping profile required (folijet)', { tags: [TestTypes.criticalPath, DevTeams.folijet] }, () => {
+    FieldMappingProfiles.checkListOfExistingProfilesIsDisplayed();
+    FieldMappingProfiles.openNewMappingProfileForm();
+    FieldMappingProfiles.checkNewMappingProfileFormIsOpened();
+
+    NewMappingProfile.addName(mappingProfileName);
+    NewMappingProfile.addIncomingRecordType('EDIFACT invoice');
+    NewMappingProfile.addFolioRecordType('Invoice');
+    NewMappingProfile.saveProfile();
```

Comment:
I tried to run this autotest and it fails because of the click of the button "Save as profile and Close"... I fails at every check because there is no click on the button. it can continue passing the test after my click...

Reply:
I fixed test